src/app/static/src/app/store/filteredData/getters.ts

```diff
@@ -2,27 +2,52 @@ import {RootState} from "../../root";
 import {DataType, FilteredDataState, SelectedChoroplethFilters, SelectedFilters} from "./filteredData";
 import {IndicatorRange, Indicators, IndicatorValues} from "../../types";
 import {interpolateCool, interpolateWarm} from "d3-scale-chromatic";
-import {FilterOption} from "../../generated";
+import {FilterOption, NestedFilterOption} from "../../generated";
+
+const sexFilterOptions = [
+    {id: "both", name: "both"},
+    {id: "female", name: "female"},
+    {id: "male", name: "male"}
+];
 
 export const getters = {
     selectedDataFilterOptions: (state: FilteredDataState, getters: any, rootState: RootState, rootGetters: any) => {
         const sapState = rootState.surveyAndProgram;
+        const regions = getters.regionOptions;
         switch(state.selectedDataType){
             case (DataType.ANC):
-                return sapState.anc ? sapState.anc.filters : null;
+                return sapState.anc ?
+                    {
+                        ...sapState.anc.filters,
+                        regions,
+                        sex: undefined,
+                        surveys: undefined,
+                    } : null;
             case (DataType.Program):
-                return sapState.program ? sapState.program.filters : null;
+                return sapState.program ?
+                    {
+                        ...sapState.program.filters,
+                        regions,
+                        sex: sexFilterOptions,
+                        surveys: undefined
+                    } : null;
             case (DataType.Survey):
-                return sapState.survey ? sapState.survey.filters : null;
+                return sapState.survey ?
+                    {
+                        ...sapState.survey.filters,
+                        regions,
+                        sex: sexFilterOptions
+                    } : null;
             default:
                 return null;
         }
     },
     regionOptions: (state: FilteredDataState, getters: any, rootState: RootState, rootGetters: any) => {
         const shape = rootState.baseline && rootState.baseline.shape ? rootState.baseline.shape : null;
+        //We're skipping the top level, country region as it doesn't really contribute to the filtering
         return shape && shape.filters &&
                         shape.filters.regions &&
-                        (shape.filters.regions as any).options ? (shape.filters.regions as any).options : [];
+                        (shape.filters.regions as any).options ? (shape.filters.regions as any).options : null;
```

Comment:
Looks like this cast to `any` is needed because the generated types still think `shape.filters.regions` is an array - just need to regenerate types I think?

Reply:
Types regenerated from https://github.com/mrc-ide/hintr/pull/44